packages/front-end/components/Tags/Tag.tsx

```diff
@@ -1,32 +1,45 @@
 import React from "react";
+import { MarginProps } from "@radix-ui/themes/dist/cjs/props/margin.props";
 import { useDefinitions } from "@/services/DefinitionsContext";
-import Badge from "@/components/Badge";
+import Badge from "@/components/Radix/Badge";
 
-interface Props {
+export const TAG_COLORS = [
+  "blue",
+  "teal",
+  "pink",
+  "orange",
+  "lime",
+  "gray",
+  "gold",
+] as const;
+export type TagColor = typeof TAG_COLORS[number];
+const isTagColor = (x: TagColor | string): x is TagColor =>
+  TAG_COLORS.includes(x as TagColor);
+
+type Props = {
   tag: string;
   color?: string;
   description?: string;
   skipMargin?: boolean;
-}
+} & MarginProps;
 
 export default function Tag({ tag, color, description, skipMargin }: Props) {
   const { getTagById } = useDefinitions();
   const fullTag = getTagById(tag);
 
   const displayTitle = description ?? fullTag?.description ?? "";
-  const displayColor = color ?? fullTag?.color ?? "#029dd1";
+
+  const tagColor = color ?? fullTag?.color ?? "blue";
+  // If a tag is still using a hex code color, we'll default to blue
+  const displayColor = isTagColor(tagColor) ? tagColor : "blue";
```

Comment:
This is too aggressive of a change.  A lot of people rely on tag colors to differentiate between types of tags.

If we want to fully move to the Radix badge component, I think we need to:
1. Allow the full range of Radix colors, not just a few handpicked ones
2. Migrate existing hex colors to the "closest" Radix color

At the very least, we should make sure all of the old pre-selected `tagColors` values map to similar Radix colors.

Finding the "closest" color is not trivial.  There's a whole wikipedia page on this topic - https://en.wikipedia.org/wiki/Color_difference .  Basically need to compute the distance between the hex and each of the Radix colors and then take the one with the smallest distance.  Depending on how perceptually accurate we want the distance to be, there's a whole bunch of different formulas to choose from.